src/client/src/apps/MainRoute/widgets/workspace/workspaceHeader/ToggleView.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import { NavItem } from './styled'
+import { TileView } from './types'
+import { ChartIcon } from 'rt-components'
+
+interface Props {
+  tileView: TileView
+  currency: string
+}
+
+const ToggleView: React.FC<Props> = ({ tileView, currency }) => {
+  const isAnalyticsViewActive = TileView.Analytics === tileView
+  const goToView = isAnalyticsViewActive ? TileView.Normal : TileView.Analytics
+  const currentView = isAnalyticsViewActive ? TileView.Normal : TileView.Analytics
```

Comment:
i think you just need the goToView here?

Reply:
Oh there is an error there! thx @ianchan035 👍